manifests/params.pp

```diff
@@ -7,13 +7,20 @@
 #
 class solr::params {
 
-  $jetty_home    = '/usr/share/jetty'
+  if $::lsbdistcodename == 'trusty' {
+    $jetty_home    = '/usr/share/jetty'
+  } else {
+    $jetty_home    = '/usr/share/jetty8'
+  }
```

Comment:
is this test correct ? for precise, xenial that jetty_home is /usr/share/jetty8 and only /usr/share/jetty on trusty

It may be better to have a switch/case selector here rather than if/else blocks